internal/cmdliveinit/cmdliveinit.go

```diff
@@ -45,13 +45,15 @@ type KptInitOptions struct {
 	Name        string // Inventory object name
 	namespace   string // Inventory object namespace
 	InventoryID string // Inventory object unique identifier label
+	Quiet       bool   // Output message during initialization
```

Comment:
Do we need this flag ? Because if we add once it live forever with v1. Why not just print the output and not provide this option for now? We can add it in future if there is a need.

Reply:
It's needed for migrate command. It's also compatible with `master`.

Reply:
I am fine with it if you are convinced that it is needed.